docs/source/index.rst

```diff
@@ -29,63 +29,41 @@ Daml Documentation
    :hidden:
    :caption: Writing Daml
 
-   daml/intro/0_Intro.rst
-   daml/reference/index
-   daml/stdlib/index
+   writing-daml
    daml/troubleshooting
-   /daml/patterns
 
 .. toctree::
    :titlesonly:
    :maxdepth: 2
    :hidden:
    :caption: Building applications
 
-   app-dev/app-arch
-   app-dev/bindings-ts/index
-   json-api/index
-   Daml Script <daml-script/index>
-   Daml REPL <daml-repl/index>
-   upgrade/index
-   app-dev/authorization
-   app-dev/ledger-api
-   app-dev/command-deduplication
-   triggers/index
-   tools/trigger-service/index
-   tools/auth-middleware/index
+   building-applications
```

Comment:
@moritzkiefer-da I added it to index.rst, here. And ran `bazel build //docs:sphinx-pdf` and didn't get local errors

Reply:
That’s the one for the html version not the one for the pdf version. Not sure why you don’t get errors. I can reproduce them locally. Are you sure you didn’t misread the output?

Reply:
pushed a fix